README.md

```diff
@@ -44,6 +44,7 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
             "Jinja",                // Enables Jinja (Flask) Template debugging
             "FixFilePathCase",      // See FIX_FILE_PATH_CASE in wrapper.py
             "DebugStdLib"           // Whether to enable debugging of standard library functions
+            "UseSourceReferences"   // Whether to use source references (making `pathMappings` optional)
```

Comment:
Is there any particular scenario that you have in mind when the user might want to *not* specify this? In VS, at least, all debuggers that support this functionality (e.g. JS), do so unconditionally.

Reply:
I couldn't think of any reason, which is why I was planning on making this the default in VSC.

Reply:
Will remove this.